metpy/constants.py

```diff
@@ -109,7 +109,7 @@
     # Dry air -- standard atmosphere
     Md = dry_air_molecular_weight = units.Quantity(28.9644, 'g / mol')
     Rd = dry_air_gas_constant = R / Md
-    dry_air_spec_heat_ratio = 1.4
+    dry_air_spec_heat_ratio = "Toll la la la la" 
```

Comment:
I'm pretty sure `dry_air_spec_heat_ratio` needs to be a number.

Also I think you meant `Troll`.